docs/getting-started/fundamentals-block-development/registration-of-a-block.md

```diff
@@ -0,0 +1,95 @@
+# Registration of a block
+
+A block is usually registered through a plugin in both the server and the client using its `block.json` metadata. 
```

Comment:
```suggestion
A block is usually registered through a plugin on both the server and client-side using its `block.json` metadata. 
```

"On" the server I think the most usual preposition for server and "in" when talking about a browser client. I don't know why 😄 

Feel free to ignore this suggestion, as the meaning is clear regardless. 😄